download/download_test.go

```diff
@@ -663,6 +766,13 @@ func (t *testServer) handle(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if (t.opaVendorMediaTypeEnabled) || (t.longPoll && !notModified) {
```

Comment:
Could we simplify this ? Also `!notModified` seems a bit weird so we can change that too.

```go
if  !notModified {
    if t.longPoll {
          w.Header().Add("Content-Type", "application/vnd.openpolicyagent.bundles")
    } else {
         w.Header().Add("Content-Type", "application/gzip")  
    }
} 
```

Reply:
I try to introduce `contentTypeShouldBeSend` which is better naming. and i a remove `opaVendorMediaTypeEnabled`. 
Let me known if it's ok for you.